gravity-forms/gw-limit-multiselect.js

```diff
@@ -63,6 +63,12 @@ function limitMultiSelect( $select, max ) {
 			.find( 'option:not(:checked)' )
 			.prop( 'disabled', selectedCount >= max )
 			.trigger( 'chosen:updated' );
+
+		// For Tom Select (GP Advanced Select)
+		// Update "1" to the ID of your Multi Select field.
+		if ( $( '#input_GFFORMID_1-ts-dropdown' ) ) {
+			$( '#input_GFFORMID_1-ts-dropdown .option' ).removeAttr( 'data-selectable' );
+		}
```

Comment:
Curious, if you unselect choices (to get below the max), do the disabled choices get re-enabled?

This snippet reads like even without Tom Select, they'd stay disabled.

Reply:
https://secure.helpscout.net/conversation/2301130622/52247/#thread-6885586156